docs/commands.md

```diff
@@ -1606,7 +1607,7 @@ Example: http://{{region}}.eureka.url.to.use:8080/eureka-server/v2
 Using {{region}} will make Spinnaker use AWS regions in the hostname to access discovery so that you can have discovery for multiple regions.
  * `--edda`: The endpoint Edda is reachable at. Edda is not a hard dependency of Spinnaker, but is helpful for reducing the request volume against AWS. See https://github.com/Netflix/edda for more information.
  * `--no-validate`: (*Default*: `false`) Skip validation.
- * `--regions`: The AWS regions this Spinnaker account will manage.
+ * `--regions`: (*Default*: `[]`) The AWS regions this Spinnaker account will manage.
```

Comment:
Not this line, but I believe there's a halyard change in response to https://github.com/spinnaker/clouddriver/pull/1544.